restdocs-openapi/src/main/kotlin/com/epages/restdocs/openapi/MockMvcRestDocumentationWrapper.kt

```diff
@@ -83,21 +103,19 @@ object MockMvcRestDocumentationWrapper {
         return document(identifier, description, null, privateResource, snippets = *snippets)
     }
 
+    @JvmStatic
+    fun resource(): ResourceSnippetDetails {
```

Comment:
I think I would rather call this method `resourceDetails` - what do you think?

Reply:
I will make the change.